packages/contracts/evm-contracts/contracts/Proxy/NativeProxy.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.7;
+
+import "../ERC1967.sol";
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+
+/// @title Proxy
+/// @dev Proxy contract mostly based on https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/Proxy.sol
+contract NativeProxy is ERC1967 {
```

Comment:
This is a proxy for NativeNftSale contract right? I would propose changing the name to NativeNftSaleProxy, so it's more explanatory and consistent with the Proxy naming